### PR TITLE
Corey/tie participants

### PR DESCRIPTION
**Pull request by PoLoMoTo**
Add frontend and backend support for manually tying two participants together who were not caught during signup.  Cannot tie participants who have ever participated in an event together.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -217,6 +217,21 @@ app.get('/participant/:id', authCheck, async (req, res) => {
   res.render('participants/singleParticipant', {participants: await db.queryParticipantByID(req.params.id), event: (await db.queryEventByID(req.params.id))[0]})
 });
 
+/**
+ * Renders the participant list to tie with the selected participant
+ */
+app.get('/participants/tie/:id', async (req, res) => {
```
Comment (JordanH821) [inline server.js:223]: add authcheck

File: server.js [hunk 2]
```diff
@@ -217,6 +217,21 @@ app.get('/participant/:id', authCheck, async (req, res) => {
...
+  res.render('participants/tieParticipants', {selected: (await db.queryParticipantByID(req.params.id))[0], participants: await db.queryParticipantsByNotID(req.params.id)})
+});
+
+/**
+ * Ties two participants together and renders the all participants view with a success alert
+ */
+app.get('/participants/tie/:id/:idwith', async (req, res) => {
```
Comment (JordanH821) [inline server.js:230]: add authcheck

File: server.js (hunk 1 above)
Comment (PoLoMoTo) [reply server.js]: Done

File: server.js (hunk 2 above)
Comment (PoLoMoTo) [reply server.js]: Done

**Pull request merged**